.idea/vcs.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
```

Comment:
Could you uncommit this vcs.xml file? For some reason git is making changes to this file, Mack's PR added this file to the .gitignore because it seems like it is specific to the local environment.

Reply:
added the file to .gitignore and pushed